org.eclipse.scanning.api/src/org/eclipse/scanning/api/event/EventConstants.java

```diff
@@ -1,109 +1,109 @@
-	 * reliably and behaves like a set. The user interface sorts the status objects by
-	 * date submitted (usually) when the user looks at the queue.
-	 */
-	public static final String STATUS_SET = "org.eclipse.scanning.status.set";
-
-	/**
-	 * Topic used to tell UI users that a give consumer will be going down.
-     * It is usually better to use your own topic rather than the default.
-	 */
-	public static final String ADMIN_MESSAGE_TOPIC = "org.eclipse.scanning.consumer.administratorMessage";
-
-    /**
-     * The topic used for requests for device information.
-     */
-	public static final String DEVICE_REQUEST_TOPIC      = "org.eclipse.scanning.request.device.topic";
-
-    /**
-     * The topic used for responses to requests for device information.
-     */
-	public static final String DEVICE_RESPONSE_TOPIC      = "org.eclipse.scanning.response.device.topic";
-
-	/**
-	 * A topic on which the values of all scannables should publish. This can happen quite frequently however
-	 * not at a rate that JMS should not be able to handle providing the message is kept small.
-	 */
-	public static final String POSITION_TOPIC              = "org.eclipse.scanning.request.position.topic";
-
-	/**
-     * The topic used for positioner requests. This can be used to set the position of one or more scannables.
-     */
-	public static final String POSITIONER_REQUEST_TOPIC      = "org.eclipse.scanning.request.positioner.topic";
-
-    /**
-     * The topic used for positioner requests.
-     */
-	public static final String POSITIONER_RESPONSE_TOPIC      = "org.eclipse.scanning.response.positioner.topic";
-
-	/**
-	 * The is the topic for requests to acquire data from a detector.
-	 */
-	public static final String ACQUIRE_REQUEST_TOPIC = "org.eclipse.scanning.request.acquire.topic";
-
-	/**
-	 * The is the topic for responses to acquire data from a detector.
-	 */
-	public static final String ACQUIRE_RESPONSE_TOPIC = "org.eclipse.scanning.response.acquire.topic";
-
-	/**
-	 * When the user sets up the axes, an AxisConfiguration object will be broadcast on this event.
-	 */
-	public static final String AXIS_CONFIGURATION_TOPIC      = "org.eclipse.scanning.axis.configuration.topic";
-
-}
+/*-
+ *******************************************************************************
+ * Copyright (c) 2011, 2016 Diamond Light Source Ltd.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    Matthew Gerring - initial API and implementation and/or initial documentation
+ *******************************************************************************/
+package org.eclipse.scanning.api.event;
+
+
+/**
+ * This class represents
+ *
+ * @author Matthew Gerring
+ */
+public interface EventConstants {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1214)